### PR TITLE
Common style classes

### PR DESCRIPTION
**Pull request by Shehanka**

### CODE REVIEW

**Review by Bawanthathilan — DISMISSED** [review]

File: src/app/components/events/events.component.html [hunk 1]
```diff
@@ -1,9 +1,6 @@
+<p class="page-title">Upcoming Event</p>
```
Comment (Shehanka) [inline src/app/components/events/events.component.html:1]: @Bawanthathilan in here I used the same CSS class which I used for about page but font sizes are different?
Screenshot 2021-04-28 at 19 04 17
Screenshot 2021-04-28 at 19 04 25

Comment (Bawanthathilan) [reply src/app/components/events/events.component.html:1]: font family are different 🙄

Comment (Shehanka) [reply src/app/components/events/events.component.html:1]: @Bawanthathilan can you fix that?

**Comment by Shehanka** [comment]
@Safnaj can u help me on this?

**Review by Bawanthathilan — DISMISSED** [review]

**Comment by Shehanka** [comment]
@Bawanthathilan done ?

**Comment by Bawanthathilan** [comment]
> @Bawanthathilan done ?

yes

**Review by Bawanthathilan — APPROVED** [review]

**Pull request merged**